docs/source/example_input/ionization_script.py

```diff
@@ -147,12 +148,17 @@ def dens_func( z, r ) :
     # Configure the moving window
     sim.set_moving_window( v=v_window )
 
-    # Add a diagnostics
+    # Add diagnostics
     sim.diags = [
                 FieldDiagnostic( diag_period, sim.fld, comm=sim.comm ),
                 ParticleDiagnostic( diag_period,
                     {"electrons from N": elec_from_N, "electrons": elec},
-                    comm=sim.comm )
+                    comm=sim.comm ),
+                # Since rho from `FieldDiagnostic` is 0 almost everywhere
+                # (neutral plasma), it is useful to see the charge density
+                # of individual particles
+                ParticleDensityDiagnostic( diag_period, sim,
```

Comment:
Maybe we should call it ParticleChargeDensityDiagnostic. Although this name is very long, it should be very clear to the user that the output density is charge dependent.